index.html

```diff
@@ -5,8 +5,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel= "stylesheet" href="styles/index.css">
 </head>
 <body>
-    
+    <div class="group-container">
+        <div class="weather-report">
+            <header>
+                <h1>🌤City Weather Report🌨</h1>
+                <p id="city" onchange="update();">For the beautiful city of <strong id="city-name">Seattle</strong> </p>
+            </header>
+        </div>
+        
+        <div class="temperature-container">
```

Comment:
Div tags will work, but I recommend using semantic HTML when possible - for example, section tags would work well here to divide the different containers.